packages/fxa-auth-server/lib/senders/templates/partials/automatedEmailChangePassword.html

```diff
@@ -0,0 +1,8 @@
+<tr style="page-break-before: always">
+  <td border="0" cellpadding="0" cellspacing="0" height="100%" width="100%">
+    <p class="secondary" style="font-family: sans-serif; font-weight: normal; margin: 0 0 12px 0; text-align: center; color: #737373; font-size: 11px; line-height: 18px; width: 310px !important; word-wrap:break-word">
```

Comment:
The old templates often had a weird `<br />` before this `<p>`. It turns out the `<br />` was there to "fix" problems caused by unclosed tags earlier in the document. Now that we have well-formed markup, I was able to get rid of it.